modules/ROOT/pages/policy-mule4-tokenization.adoc

```diff
@@ -21,29 +21,41 @@ endif::[]
 
 |===
 
-[NOTE]
---
-This policy is designed to work only for Anypoint Runtime Fabric deployments.
---
+Tokenization is the process of tokenizing a value or piece of information, which might be considered sensitive data, into the token that can be mapped back to the original value. 
+
+This tokenized information replaces the original value with the token, thus minimizing the risk of matching it to the original value, in case a third party accesses the data. For example, this policy may be used to tokenize credit card numbers and map them to completely different ones from the originals.
+
+If your API contains sensitive data, the tokenization policy is a highly effective way to protect it. Examples of sensitive information that are suitable for tokenization protection include credit card numbers, personally identifiable information (PII), protected health information (PHI), or any  other information that is deemed sensitive.
 
-== Tokenization
 
-Tokenization is the process of tokenizing a value or piece of information (presumably sensitive data) into another one (i.e., the token), that can be mapped back onto the original value. +
-This tokenized information, replaces the original value with the token, thus minimizing the risk of matching it to the raw value in case someone happened to read it. For instance, it may be used to tokenize credit card numbers and map them to completely different ones from the originals.
 
 == Prerequisites
 
-* To create and apply policies you need the *Manage Policies* permission.
-* You need to have a tokenization service up and running.
-** To configure and see the tokenization service, you need the *Manage Tokenization Services* permission. +
-See xref:anypoint-security::tokenization-example.adoc[Configure and Use Tokenization] for more information.
+To apply and manage the Tokenization policy, you must have:
+
+* Ontained the *Manage Policies* permission.
```

Comment:
Could not find `Ontained` as a word in google, is this a typo?

Reply:
Fixed! Thanks!